analysis/import/SourceCodeParser/src/test/java/de/maibornwolff/codecharta/importer/sourcecodeparser/oop/infrastructure/antlr/java/AntlrAbstractSyntaxTreeTest.java

```diff
@@ -0,0 +1,115 @@
+package de.maibornwolff.codecharta.importer.sourcecodeparser.oop.infrastructure.antlr.java;
```

Comment:
Is this test really necessary ? As far as I can see it checks Antlr itself and should be out of our testing scope. Is there a reason why this test is needed ?